README.md

```diff
@@ -145,6 +145,13 @@ check docs](docs/healthchecks.md) for more information on how to use it.
 In Rails applications, the application will be configured to send JSON-formatted
 logs to `STDOUT` and unstructured logs to `STDERR`.
```

Comment:
Ooh we should update this in https://github.com/alphagov/govuk_app_config/pull/300 /cc @yuetylauiris